### DISCUSSION TITLE
Organic Maps for iOS 6?

### DISCUSSION THREAD

**Discussion by rivciks** [organicmaps/organicmaps#5038] [Ideas]
Hi guys,
I am curios, is it possible to make Organic Maps also for iOS 6 available? It would make older iPhones solid GPS-devices since those devices are  very robust and handy and otherwise are not really usable  as daily devices.

**Comment by biodranik** [comment]
In theory, if you have a lot of time, you can try it ) But isn't it better to earn some money at this time and buy a newer device?

**Comment by rivciks** [comment]
Yes, but how? How to compile it for iOS 6? If someone could just guide me through it step by step...

Newer smartphones with their huge displays are not acceptable since they have to endure a lot of pressure on them and do easily break. I do a lot of hiking and other activities and carry the smartphone in my pocket. For those purposes the iPhone 4/iPhone 4s on iOS 6 with their smaller displays were ideal. Nonetheless, even Steve Jobs pointed out that the first models of  iPhone were designed specially to fit in the pocket.  I would very much like to have the Organic Maps on the iPhone 4/4s with iOS 6 since those models as I mentioned above are very robust, small and handy and are ideal to use as a GPS-device for outdoor activities.

**Reply by hb0nd** [reply]
To build for iOS 6, you need a Mac (only macs work) & Xcode. In Xcode you can set the build target to iOS 6. However, it's unlikely it'll build without significant patching work. 
However, Apple also makes using manually built apps annoyingly difficult, you need to sign up as a developer then constantly re-build as the builds expire (at least for newer iOS)

Sadly, getting apps to build for old/end-of-life hardware is often just not worth the significant effort & tiny user base. Maybe consider getting an iPhone SE 2016 (similar size, very cheap, runs iOS 15.7)?

**Reply by rivciks** [reply]
In other circumstances I would gladly choose a newer phone as a Gps-device but... I do a lot of mountaineering and often quite a pressure is applied on the phone; you wouldn't believe it but even 0.5 inches do matter A LOT. There is a simple rule of thumb: the smaller the display, the more freedom of movement you have. I was even thinking of using a Samsung Galaxy Pocket but since that device had a very poor GPS reception, I had to abandon that thought entirely. 

I also do not need to worry about expiring certificates and other inconveniences since I have a jail-broken iOS 6 and am able just to  sideload  any IPA directly into my phone. So it wouldn't be an issue. 
I also could get my hands on Mac and Xcode and if somebody just guided me through it and showed me how to  proceed with the building and with the patching work, I would gladly take on the task. Thank You.
